index.html

```diff
@@ -62,10 +62,10 @@ <h2>$600</h2>
         <div>BASS TROMBONE</div>
         <h2>$900</h2>
         <ol>
-            <li>Lorem ipsum</li>
-            <li>Lorem ipsum</li>
-            <li>Lorem ipsum dolor</li>
-            <li>Lorem ipsum</li>
+            //<li>Lorem ipsum</li>
+            //<li>Lorem ipsum</li>
+            //<li>Lorem ipsum dolor</li>
+            //<li>Lorem ipsum</li>
```

Comment:
Good jobs so far. But your code needs few changes.
Please do not use unused code as comment.

Reply:
You did great work, however, there are some changes that you still need to perform before going to the next step.
I suggest you use polite and descriptive suggestions, like: `I think it would be better to...`

Example:

`Good job so far, but your code needs a few changes to be completed, Kindly, I suggest you remove all commented code in your code base to make your project look more professional and to follow best practices. 😄 `

Reply:
As requested by your previous reviewer, please be a bit more expressive in your suggestions as it will make more meaning to your reader. 👀 

- [ ] Kindly consider giving a reason for your suggestion as this will make your suggestions more meaningful. 
In your case you could say :
`Please do not use unused code as comment because it makes your code messy and difficult to read` ✔️